docs/content/AutoGPT/configuration/search.md

```diff
@@ -13,25 +13,25 @@ Create your project:
     page and clicking *New Project*
 4. Give it a name and click *Create*
 5. Set up a custom search API and add to your .env file:
-    5. Go to the [APIs & Services Dashboard](https://console.cloud.google.com/apis/dashboard)
-    6. Click *Enable APIs and Services*
-    7. Search for *Custom Search API* and click on it
-    8. Click *Enable*
-    9. Go to the [Credentials](https://console.cloud.google.com/apis/credentials) page
-    10. Click *Create Credentials*
-    11. Choose *API Key*
-    12. Copy the API key
-    13. Set it as the `GOOGLE_API_KEY` in your `.env` file
-14. [Enable](https://console.developers.google.com/apis/api/customsearch.googleapis.com)
+    1. Go to the [APIs & Services Dashboard](https://console.cloud.google.com/apis/dashboard)
+    2. Click *Enable APIs and Services*
+    3. Search for *Custom Search API* and click on it
+    4. Click *Enable*
+    5. Go to the [Credentials](https://console.cloud.google.com/apis/credentials) page
+    6.  Click *Create Credentials*
+    7.  Choose *API Key*
+    8.  Copy the API key
+    9.  Set it as the `GOOGLE_API_KEY` in your `.env` file
+6.  [Enable](https://console.developers.google.com/apis/api/customsearch.googleapis.com)
     the Custom Search API on your project. (Might need to wait few minutes to propagate.)
     Set up a custom search engine and add to your .env file:
-    15. Go to the [Custom Search Engine](https://cse.google.com/cse/all) page
-    16. Click *Add*
-    17. Set up your search engine by following the prompts.
+    1.  Go to the [Custom Search Engine](https://cse.google.com/cse/all) page
+    2.  Click *Add*
+    3.  Set up your search engine by following the prompts.
         You can choose to search the entire web or specific sites
-    18. Once you've created your search engine, click on *Control Panel*
-    19. Click *Basics*
-    20. Copy the *Search engine ID*
-    21. Set it as the `CUSTOM_SEARCH_ENGINE_ID` in your `.env` file
+    4.  Once you've created your search engine, click on *Control Panel*
+    5.  Click *Basics*
+    6.  Copy the *Search engine ID*
+    7.  Set it as the `CUSTOM_SEARCH_ENGINE_ID` in your `.env` file
```

Comment:
Nit: inconsistent amount of space after numbering

Reply:
set all numbers to 1. and it will allow editing without renumbering

Reply:
I fixed spacing and changed all numbers to `1.`